src/features/store/components/product-view.tsx

```diff
@@ -0,0 +1,230 @@
+      price: 200,
+      image: "/placeholder.svg?height=300&width=300",
+    },
+    {
+      id: 3,
+      name: "Coffee Beans",
+      seller: "Loan Farmer Limited",
+      price: 200,
+      image: "/placeholder.svg?height=300&width=300",
+    },
+    {
+      id: 4,
+      name: "Coffee Beans",
+      seller: "Loan Farmer Limited",
+      price: 200,
+      image: "/placeholder.svg?height=300&width=300",
+    },
+  ]
+
+  useEffect(() => {
+    postCall<{ data: ProductType }>(ENDPOINTS.GETPRODUCTBYID, { id: productId }).then((resp) => {
+      if (resp && resp.data) {
+        setProductDetails(resp.data.data);
+      }
+    });
+  }, []);
+
+  const decreaseQuantity = () => {
+    if (quantity > 1) {
+      setQuantity(quantity - 1)
+    }
+  }
+
+  const increaseQuantity = () => {
+    setQuantity(quantity + 1)
+  }
+
+  return (
+    <div className="container mx-auto px-4 py-8">
+      {/* Breadcrumb */}
+      <nav className="flex mb-8 text-sm">
+        <Link to="/shop" className="text-gray-500 hover:text-gray-700">
+          Shop
+        </Link>
+        <span className="mx-2 text-gray-400">/</span>
+        <Link to="/shop/beans" className="text-gray-500 hover:text-gray-700">
+          Beans
+        </Link>
+        <span className="mx-2 text-gray-400">/</span>
+        <span className="text-gray-700">{productDetails?.name}</span>
+      </nav>
+
+      {/* Product Details */}
+      <div className="grid grid-cols-1 md:grid-cols-2 gap-8 mb-16">
+        {/* Product Images */}
+        <div className="space-y-4">
+          <div className="bg-gray-100 rounded-lg overflow-hidden">
+            <img
+              src={productImages[selectedImage] || "/placeholder.svg"}
+              alt="White Beans"
+              width={600}
+              height={600}
+              className="w-full h-auto object-contain"
+            />
+          </div>
+
+          {/* Thumbnails */}
+          <div className="grid grid-cols-4 gap-2">
+            {productImages.map((_, index) => (
+              <Button
+                key={index}
+                onClick={() => setSelectedImage(index)}
+                className={`bg-gray-100 rounded-lg overflow-hidden border-2 ${selectedImage === index ? "border-green-500" : "border-transparent"
+                  }`}
+              >
+                <img
+                  src={DeliveryBgImg || "/placeholder.svg"}
+                  alt={`Thumbnail ${index + 1}`}
+                  width={100}
+                  height={100}
+                  className="w-full h-auto object-contain"
+                />
+              </Button>
+            ))}
+          </div>
+        </div>
+
+        {/* Product Info */}
+        <div>
+          <h1 className="text-3xl font-bold mb-4">{productDetails?.name}</h1>
+
+          <div className="flex items-center mb-2">
+            <ShoppingCart className="h-5 w-5 text-gray-500 mr-2" />
+            <span className="text-gray-600">Sold by Loan Farmer Limited</span>
```

Comment:
As you may know. all this data/information is not suppose to be hardcoded. Is this temporary?